zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -425,6 +425,20 @@ private void broadcast(String noteId, Message m) {
     }
   }
 
+  private void broadcastToUser(String noteId, String user, Message m) {
```

Comment:
i think there's already pretty similar `multicastToUser` function in [here](https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java#L449) that does pretty much the same. so in longer run we may either combine them or keep the same naming with different parameters i think